docs/community.rst

```diff
@@ -1,15 +1,15 @@
 Community
 ---------
 
-You can check out Kagi on GitHub at https://github.com/justinmayer/kagi/.
+You can check out Kagi on GitHub at https://GitHub.com/justinmayer/kagi/.
```

Comment:
I think the global search-and-replace affected the domain name within links, which should probably remain lower-case.